huaweicloud/provider.go

```diff
@@ -731,9 +732,9 @@ func Provider() *schema.Provider {
 			"huaweicloud_dms_rocketmq_topic":          dms.ResourceDmsRocketMQTopic(),
 			"huaweicloud_dms_rocketmq_user":           dms.ResourceDmsRocketMQUser(),
 
-			"huaweicloud_dns_ptrrecord": ResourceDNSPtrRecordV2(),
-			"huaweicloud_dns_recordset": ResourceDNSRecordSetV2(),
-			"huaweicloud_dns_zone":      ResourceDNSZoneV2(),
+			"huaweicloud_dns_ptrrecord": dns.ResourceDNSPtrRecordV2(),
+			"huaweicloud_dns_recordset": dns.ResourceDNSRecordSetV2(),
+			"huaweicloud_dns_zone":      dns.ResourceDNSZoneV2(),
```

Comment:
it's better to remove the **v2** suffix in both file name and function name, except for `huaweicloud_dns_recordset` that will be refactored later.